src/test/kotlin/MicronautAuthenticationTest.kt

```diff
@@ -0,0 +1,221 @@
+package com.cjbooms.fabrikt.generators
```

Comment:
**issue:** Your test files aren't actually at the location where their path indicates they should be.

Reply:
done thanks to you